src/Microsoft.DocAsCode.Build.TableOfContents/BuildTocDocument.cs

```diff
@@ -99,5 +99,19 @@ private void BuildCore(TocItemViewModel item, FileModel model, IHostService host
                 }
             }
         }
+
+        #region ISupportIncrementalBuild Members
+
+        public bool CanIncrementalBuild(FileAndType fileAndType)
+        {
+            return true;
+        }
+
+        public string GetIncrementalContextHash()
+        {
+            return null;
```

Comment:
I think `IncrementalContextHash == null` means the step is stateless and can process those changed files only. @ansyral, correct?